module/zfs/zfs_chksum.c

```diff
@@ -195,19 +226,36 @@ chksum_benchit(chksum_stat_t *cs)
 	chksum_run(cs, abd, ctx, 3, &cs->bs16k);
 	chksum_run(cs, abd, ctx, 4, &cs->bs64k);
 	chksum_run(cs, abd, ctx, 5, &cs->bs256k);
+
+	/* check if we ran on a slow cpu */
+	if (limit == LIMIT_INIT) {
+		if (cs->bs1k < 300) {
```

Comment:
I think leaving this hard coded is fine, but let's at least add a #define to specify it for clarity.  Maybe:

```
/* 
#define LIMIT_PERF_MBS 300
```

Reply:
```
/* limit benchmarking to max 256KiB, when EdonR is slower then this: */
#define LIMIT_PERF_MBS 300
```